docs/quick_start.md

```diff
@@ -14,6 +14,7 @@ Ensure your machine satisfies the following requirements:
 
 1. Setup the environment
 ```bash
+export OPEN_AEA_IPFS_ADDR="/dns/registry.autonolas.tech/tcp/443/https"
```

Comment:
See [this](https://trello.com/c/xLTJZa4z/758-set-default-remote-registry-to-be?filter=member:dvilelaf)

Reply:
Can this not be provided as part of the init command @angrybayblade ?